Extensions/ShareTo/ShareViewController.swift

```diff
@@ -63,13 +64,35 @@ class ShareViewController: UIViewController {
         setupNavBar()
         setupStackView()
 
+        guard let shareItem = shareItem else { return }
+
+        self.setupRows()
+
+        switch shareItem {
+        case .shareItem(let item):
+            self.pageInfoRowUrlLabel?.text = item.url
+            self.pageInfoRowTitleLabel?.text = item.title
+        case .rawText(let text):
+            self.pageInfoRowTitleLabel?.text = "“" + text + "”"
```

Comment:
Cool unicode open/close quotes. Need to consider non-english quotes though, will investigate what to use here.